nb-configuration.xml

```diff
@@ -15,8 +15,7 @@ Any value defined here will override the pom.xml file value but is only applicab
 -->
         <org-netbeans-modules-javascript2-requirejs.enabled>true</org-netbeans-modules-javascript2-requirejs.enabled>
         <netbeans.checkstyle.format>true</netbeans.checkstyle.format>
-        <netbeans.compile.on.save>none</netbeans.compile.on.save>
-        <netbeans.hint.jdkPlatform>JDK_11</netbeans.hint.jdkPlatform>
+        <netbeans.compile.on.save>all</netbeans.compile.on.save>
```

Comment:
Hi @toddthetrainnut , thanks for the contribution !

If the nb-configuration.xml changes could be made on a separate PR if intended?
If I recall correctly, compile on save turned on caused issues for some developers a while back.
For the moment, we need Netbeans to validate against Java 11

Thanks,
Steve.

Reply:
I'm not the git wiz I should be, so I've added a new commit to revert the changes to nb-configuration.xml rather than amending the commit that made the change in the first place.

I believe I changed the compile-on-save because otherwise NetBeans was compiling the _entire_ JMRI project every time I ran, which took ages. I have no idea how or why the jdkPlatform line was removed, though.

Reply:
> If I recall correctly, compile on save turned on caused issues for some developers a while back.

**One** developer wanted it and turned it on. But other developers disagree. I don't think we have had a serious discussion about it on the JMRI developers list. It has both advantages and disadvantages.